content/en/docs/ambient/install/multicluster/multi-primary_multi-network/index.md

```diff
@@ -0,0 +1,440 @@
+---
+title: Install ambient multi-primary on different networks
+description: Install an Istio ambient mesh across multiple primary clusters on different networks.
+weight: 30
+keywords: [kubernetes,multicluster]
+test: yes
+owner: istio/wg-environments-maintainers
+---
+
+{{< boilerplate alpha >}}
+
+Follow this guide to install the Istio control plane on both `cluster1` and
+`cluster2`, making each a {{< gloss >}}primary cluster{{< /gloss >}} (this is currently the only supported configuration in ambient mode). Cluster
+`cluster1` is on the `network1` network, while `cluster2` is on the
+`network2` network. This means there is no direct connectivity between pods
+across cluster boundaries.
+
+Before proceeding, be sure to complete the steps under
+[before you begin](/docs/ambient/install/multicluster/before-you-begin).
+
+{{< boilerplate multi-cluster-with-metallb >}}
+
+In this configuration, both `cluster1` and `cluster2` observe the API Servers
+in each cluster for endpoints.
+
+Service workloads across cluster boundaries communicate indirectly, via
+dedicated gateways for [east-west](https://en.wikipedia.org/wiki/East-west_traffic)
+traffic. The gateway in each cluster must be reachable from the other cluster.
+
+{{< image width="75%"
+    link="arch.svg"
+    caption="Multiple primary clusters on separate networks"
+    >}}
+
+## Set the default network for `cluster1`
+
+If the istio-system namespace is already created, we need to set the cluster's network there:
+
+{{< text bash >}}
+$ kubectl --context="${CTX_CLUSTER1}" get namespace istio-system && \
+  kubectl --context="${CTX_CLUSTER1}" label namespace istio-system topology.istio.io/network=network1
+{{< /text >}}
+
+## Configure `cluster1` as a primary
+
+Create the `istioctl` configuration for `cluster1`:
+
+{{< tabset category-name="multicluster-install-type-cluster-1" >}}
+
+{{< tab name="IstioOperator" category-value="iop" >}}
+
+Install Istio as primary in `cluster1` using istioctl and the `IstioOperator` API.
+
+{{< text bash >}}
+$ cat <<EOF > cluster1.yaml
```

Comment:
This config didn't work for me (didn't have ztunnels), I changed it this way and it worked:

```
apiVersion: insall.istio.io/v1alpha1
kind: IstioOperator
spec:
  profile: ambient
  components:
    pilot:
      k8s:
        env:
          - name: AMBIENT_ENABLE_MULTI_NETWORK
            value: "true"
  values:
    global:
      meshID: mesh1
      multiCluster:
        clusterName: cluster1
      network: network1
```

The same for the second cluster.